packages/account/src/Sections/Security/ClosingAccount/account-has-balance.jsx

```diff
@@ -270,6 +284,55 @@ const AccountHasPendingConditions = ({
                         ))}
                     </Wrapper>
                 )}
+                {!!ctrader_open_positions.length && (
```

Comment:
@hamza-deriv please remove the duplication and make a reusable component for each wrapper and pass `DerivX` and `CTrader` as props

Reply:
We need to refactor this into small atomic component as this is part of `Account` package so need input from the respective team.
cc: @mahdiyeh-deriv @amina-deriv